src/go/rpk/pkg/cli/cmd/common/common_test.go

```diff
@@ -97,15 +91,13 @@ func TestDeduceBrokers(t *testing.T) {
 		name: "it should prioritize the config over the default broker addr",
 		config: func() (*config.Config, error) {
 			conf := config.Default()
-			conf.Redpanda.KafkaApi = []config.NamedSocketAddress{{
-				SocketAddress: config.SocketAddress{
-					Address: "192.168.25.87",
-					Port:    1234,
-				},
-			}}
+			conf.Rpk.KafkaApi.Brokers = []string{"192.168.25.87:1234", "192.168.26.98:9092"}
```

Comment:
Any reason to change the test here to return two brokers now instead of one?

Reply:
Just mixin' it up.